src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -233,6 +233,17 @@ public void waitForElementPresence(By by){
         WebDriverWait wait = new WebDriverWait(browser.driver, TestProperties.inst().TEST_TIMEOUT);
         wait.until(ExpectedConditions.presenceOfElementLocated(by));
     }
+
+    /**
+     * Waits for the element to appear in the page, up to the timeout specified and can
+     * return early if the expected conditions has happened
+     * @param by
+     * @param timeToImplicitlyWait
```

Comment:
You can mention that timeToImplicitlyWait is in seconds here, or rename the variable.